contracts/AppInstance.sol

```diff
@@ -6,12 +6,12 @@ import "./lib/StaticCall.sol";
 import "./lib/Transfer.sol";
 
 
-/// @title StateChannel - A generalized state channel application contract
+/// @title AppInstance - A contract that defines a state channel application
```

Comment:
So by this terminology, deploying a state channel application gives an instance of an `AppInstance`?

Why not just name it `App` then? Then it's more intuitive to have an _instance_ of an app?

Reply:
I think it's to distinguish between ambiguous meanings for "App": Instance and Definition

Reply:
Yeap what @mvanderh said, I feel like having separate `App` and a `AppDefinition` contracts is more confusing

Reply:
Fair enough